src/backend/marsha/core/api/video.py

```diff
@@ -637,8 +637,8 @@ def get_queryset(self):
         depending on the role.
         """
         user = self.request.user
+        filters = {"video__id": user.id}
```

Comment:
I don't understand this change. Can you explain me why ?

Reply:
it was initialized in the case LTI as only LTI was doing a query, now that we query in both cases, it has been moved over the "if"